exampleCourse/elements/course-element/course-element.py

```diff
@@ -9,7 +9,8 @@ def prepare(element_html, data):
 
 def render(element_html, data):
     html_params = {
-        'number': data['params']['random_number']
+        'number': data['params']['random_number'],
+        'image_url': data['options']['elements_url'] + '/course-element/clientFilesElement/block_i.png'
```

Comment:
I wonder if there's a way to do this that automatically maps the `elements_url` to include the element name folder and clientFilesElement without having to hard code it in the element in the element python code. i.e. 
'image_url': data['options']['elements_url'] + '/block_i.png'

Reply:
+1 - this would require mutating `data.options` during HTML traversal which we don't currently do for any other case, but I don't see a problem with that. Avoiding hardcoding element names in element code would make them easier to copy/fork/rename/etc.